opentelemetry-instrumentation/src/opentelemetry/instrumentation/auto_instrumentation/sitecustomize.py

```diff
@@ -27,8 +29,19 @@ def _load_distros():
 
 
 def _load_instrumentors():
+    package_to_exclude = Configuration().get("DISABLED_INSTRUMENTATIONS", [])
```

Comment:
Why default empty array?

Reply:
@lzchen , this was modified as per reviews from @ocelotl  and @owais.  [here](https://github.com/open-telemetry/opentelemetry-python/pull/1461#discussion_r540610497)  and [here](https://github.com/open-telemetry/opentelemetry-python/pull/1461#discussion_r541511014)

I got rid of if/else loop  which checks whether package_to_exclude is None